src/promptflow/promptflow/_sdk/_submitter/experiment_orchestrator.py

```diff
@@ -484,21 +484,27 @@ def set_orchestrator_terminated():
 class ExperimentNodeRun(Run):
     """Experiment node run, includes experiment running context, like data, inputs and runs."""
 
-    def __init__(self, node, experiment, node_runs, run_operations, **kwargs):
+    def __init__(self, node, experiment, context, node_runs, client, **kwargs):
         from promptflow._sdk._configuration import Configuration
 
         self.node = node
+        self.context = context
         self.experiment = experiment
         self.experiment_data = {data.name: data for data in experiment.data}
         self.experiment_inputs = {input.name: input for input in experiment.inputs}
         self.node_runs = node_runs
-        self.run_operations = run_operations
+        self.client = client
+        self.run_operations = self.client.runs
 
         timestamp = datetime.now().strftime("%Y%m%d_%H%M%S_%f")
+        self.node_context = self.context.get_node_context(
+            node.name, is_flow=node.type == ExperimentNodeType.FLOW, test=False
+        )
         # Config run output path to experiment output folder
         run_output_path = (Path(experiment._output_dir) / "runs" / node.name).resolve().absolute().as_posix()
         super().__init__(
             # Use node name as prefix for run name?
+            type=RunTypes.COMMAND if node.type == ExperimentNodeType.COMMAND else RunTypes.BATCH,
```

Comment:
Is run type of flow node and chat group node are also defined as Batch?

Reply:
Offline synced, we are not sure if chat group node has a new type for now.